CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.3.0
```

Comment:
There are typos at line 27:
"Fork configuration to enable decentralized governance (ZenDao) and change consensus parameters (consensus epoch slot time and **lenght**, **acrive** slot coefficient)" => "Fork configuration to enable decentralized governance (ZenDao) and change consensus parameters (consensus epoch slot time and **length**, **active** slot coefficient)"